olclient/common.py

```diff
@@ -30,7 +32,7 @@ def add_id(self, id_type, identifier):
 
         Args:
             id_type (unicode) - valid identifier types:
-              [u'olid', u'oclc', u'isbn_10', u'isbn_13']
+              [u'olid', u'oclc', u'isbns']
```

Comment:
@mekarpeles Shouldn’t this be `isbn` as all other ids are singular? (...even though they are lists)